src/components/molecules/SpaceEditFormNG/SpaceEditFormNG.scss

```
@@ -26,12 +37,35 @@
     width: 100%;
   }
 
-  &__capacity {
-    display: grid;
-    grid-template-columns: repeat(3, 1fr);
-    grid-template-rows: repeat(2, 1fr);
+  .AdminInput__input {
+    text-align: left;
 
-    align-items: center;
-    text-align: center;
+    &::placeholder {
+      word-break: break-word;
+    }
+  }
+
+  .AdminTextarea__input {
+    height: $textarea-height;
+  }
+
+  .SpacesDropdown {
+    .Dropdown {
+      width: 100%;
+      background-color: opaque-white(0.12);
```

Try not using `opaque-white()` and `opaque-black()` but instead copy from what they replace so that we can eventually remove them:

```suggestion
      background-color: var(--content--over-10a );
```

---

done 77985a4